master/buildbot/master.py

```diff
@@ -132,6 +133,9 @@ def create_child_services(self):
         self.metrics = metrics.MetricLogObserver()
         self.metrics.setServiceParent(self)
 
+        self.stats_service = stats_service.StatsService(self)
```

Comment:
The documentation is not entirely up to date on that part. See the cfg-statustarget.rst documentation.